reactive/kotlinx-coroutines-jdk9/test/IntegrationTest.kt

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright 2016-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.coroutines.jdk9
+
+import kotlinx.coroutines.*
+import kotlinx.coroutines.flow.flowOn
+import org.hamcrest.MatcherAssert.*
+import org.hamcrest.core.*
```

Comment:
Please avoid using `hamcrest`. There is nothing wrong with it, but all the tests don't use it and it's better to be consistent across the single code base

Reply:
Same here: should I also fix this in the original tests?

Reply:
Yes, it would be a great fix while we are at it.

Reply:
Fixed in another PR.